monitoring/shutdown-javaagent/src/test/java/com/google/cloud/runtimes/monitoring/shutdown/ShutdownReporterTest.java

```diff
@@ -0,0 +1,65 @@
+package com.google.cloud.runtimes.monitoring.shutdown;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.google.cloud.runtimes.monitoring.shutdown.log.TestLogging;
+
+import org.junit.Test;
+
+public class ShutdownReporterTest {
+
+  private TestLogging logging = new TestLogging();
+
+  @Test
+  public void doesNothingIfHeapInfoOrStackTraceNotEnabled() {
```

Comment:
Good test name. I like that it says the conditions for the test and the expected result.

Could you name the other tests in this class in a similar manner? I would like to see what the expected result is so we can verify that the tests are correct.